### ISSUE TITLE
`name-tag` no longer works in `0.5.1`

### ISSUE THREAD

**Issue by zscholl** [danmx/sigil#117]
I upgraded to `0.5.1` and I'm now unable to use the `--type name-tag` selector.

Was this deprecated?

SSH config
```
Host *.instance.by.tag
    User ssm_user
    IdentityFile ~/.sigil/temp_key
    ProxyCommand sh -c 'sigil ssh --type name-tag --target %h --port %p --pub-key "${HOME}"/.sigil/$(date +%%s)_key.pub --gen-key-pair --os-user ssm_user --profile my-profile --region us-east-1
```

Error message:
```bash
❯ ssh some.instance.by.tag
Error: unsupported target type: name-tag
Usage:
  sigil ssh [flags
```

**Comment by zscholl** [comment]
Looks like it was changed to `name` in this commit: https://github.com/danmx/sigil/commit/c8a3ca038dc93a1e8f4f0464ace5d0150c5d0667

By SemVer standards this should have resulted in a Major version bump since the API changed: https://semver.org/

**Comment by danmx** [comment]
yes, I made a mistake on making it a major version bump

**Comment by danmx** [comment]
I'll add it back and bump the minor version but remove it with next major update

**Comment by zscholl** [comment]
Okay that sounds good! Thanks for the quick response @danmx

**Comment by danmx** [comment]
fix in release [0.5.3](https://github.com/danmx/sigil/releases/tag/0.5.3)
